src/fetcher.ts

```diff
@@ -26,19 +23,20 @@ export const fetch = async (a: Alarm): Promise<string> => {
   const r = await asyncRetry<GetQueryResultsResponse>(
     async () => {
       const res = await cwl.getQueryResults({ queryId: id as string }).promise()
-      return res.status === "Running" ? thrw(new Error(res.status)) : res
+      return res.status === "Running"
+        ? console.error(new Error(res.status))
```

Comment:
I think the `thrw` method that's being replaced would actually throw an Error here

Reply:
Good call! Updated!